src/cli/subcommands/chain_cmd.rs

```diff
@@ -12,6 +12,12 @@ use super::{print_pretty_json, print_rpc_res_cids};
 
 #[derive(Debug, Subcommand)]
 pub enum ChainCommands {
+    /// Retrieves and prints messages stored in the block specified by the given CID.
+    BlockMessages {
+        #[arg(short)]
+        cid: Cid,
+    },
+
```

Comment:
Why did you implement this command? What is its purpose? If it is useful, why doesn't Lotus have a similar command?

Reply:
Right, Lotus doesn't have this command. It only uses the `Filecoin.ChainGetBlockMessages` in its `chain get-block subcommand`. I can remove it.